### PR TITLE
Update the-changelog-119.md

### PR DESCRIPTION
**Pull request by sanyuj1997**
Added description, formatting.

### CODE REVIEW

File: podcast/the-changelog-119.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
-* [MEAN.JS](http://meanjs.org/)
-* [roieki (Roie Cohen)](https://github.com/roieki)
-* [amoshaviv (Amos Haviv)](https://github.com/amoshaviv)
\ No newline at end of file
+* [MEAN.JS](http://meanjs.org/) - Open-Source Full-Stack Solution For MEAN Applications!!
```
Comment (jerodsanto) [inline podcast/the-changelog-119.md]: Let's combine these two lines into one:

>  * [MEAN.JS](http://meanjs.org/) - a full-stack JavaScript solution that helps you build fast, robust, and maintainable production web applications using MongoDB, Express, AngularJS, and Node.js.

File: podcast/the-changelog-119.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
-* [MEAN.JS](http://meanjs.org/)
-* [roieki (Roie Cohen)](https://github.com/roieki)
-* [amoshaviv (Amos Haviv)](https://github.com/amoshaviv)
\ No newline at end of file
+* [MEAN.JS](http://meanjs.org/) - Open-Source Full-Stack Solution For MEAN Applications!! 
+MEAN.JS is a full-stack JavaScript solution that helps you build fast, robust, and maintainable production web applications using MongoDB, Express, AngularJS, and Node.js. MEAN.JS will help you getting started and avoid useless grunt work and common pitfalls, while keeping your application organized. 
+
```
Comment (jerodsanto) [inline podcast/the-changelog-119.md]: we typically don't add blank lines between list items

File: podcast/the-changelog-119.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
-* [MEAN.JS](http://meanjs.org/)
-* [roieki (Roie Cohen)](https://github.com/roieki)
-* [amoshaviv (Amos Haviv)](https://github.com/amoshaviv)
\ No newline at end of file
+* [MEAN.JS](http://meanjs.org/) - Open-Source Full-Stack Solution For MEAN Applications!! 
+MEAN.JS is a full-stack JavaScript solution that helps you build fast, robust, and maintainable production web applications using MongoDB, Express, AngularJS, and Node.js. MEAN.JS will help you getting started and avoid useless grunt work and common pitfalls, while keeping your application organized. 
+
+* [roieki (Roie Cohen)](https://github.com/roieki) - GitHub profile of Roie Cohen, co-founder of meanjs.org
```
Comment (jerodsanto) [inline podcast/the-changelog-119.md]: For these we can replace the link text with part of the description, eg:

> * [GitHub profile of Roie Cohen](https://github.com/roieki) - co-founder of meanjs.org

File: podcast/the-changelog-119.md (hunk 1 above)
Comment (sanyuj1997) [reply podcast/the-changelog-119.md]: Done!! xD

File: podcast/the-changelog-119.md (hunk 2 above)
Comment (sanyuj1997) [reply podcast/the-changelog-119.md]: Removed, will keep that in mind.

File: podcast/the-changelog-119.md (hunk 3 above)
Comment (sanyuj1997) [reply podcast/the-changelog-119.md]: done as well! xD

**Comment by sanyuj1997** [comment]
Haha Yaaay!!!
my first open source contribution!!! xD

On Oct 7, 2018 7:28 PM, "Jerod Santo" <notifications@github.com> wrote:

Merged #8 <https://github.com/thechangelog/show-notes/pull/8> into master.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/thechangelog/show-notes/pull/8#event-1889016101>, or mute
the thread

**Comment by jerodsanto** [comment]
Awesome! ✊

**Pull request merged**